### ISSUE TITLE
Make `pLimit()` support an options object

### ISSUE THREAD

**Issue by Richienb** [sindresorhus/p-limit#89]
Decision: https://github.com/sindresorhus/p-limit/pull/88#discussion_r1869324258

@sindresorhus Should it still be backwards compatible?

**Comment by sindresorhus** [comment]
Yes, no point in breaking backwards compatibility for something like that.
